### PR TITLE
Update csgo.hpp 5/12/2022

### PR DESCRIPTION
**Pull request by pachara1337**

### CODE REVIEW

File: csgo.hpp [hunk 1]
```diff
@@ -1,25 +1,17 @@
 #pragma once
 #include <cstdint>
 
-// 2022-11-19 11:00:16.742708200 UTC
+// 2022-12-05 06:33:AM UTC
 
-namespace hazedumper {
-constexpr ::std::int64_t timestamp = 1668855616;
+namespace hazdumper {
```
Comment (JuanUWU69) [inline csgo.hpp:6]: you should fix this typo 😄

File: csgo.hpp [hunk 2]
```diff
@@ -117,48 +118,47 @@ constexpr ::std::ptrdiff_t dwClientState_MaxPlayer = 0x388;
...
 constexpr ::std::ptrdiff_t m_pStudioHdr = 0x2950;
-constexpr ::std::ptrdiff_t m_pitchClassPtr = 0x5235FA0;
-constexpr ::std::ptrdiff_t m_yawClassPtr = 0xDEB910;
+constexpr ::std::ptrdiff_t m_yawClassPtr = 0xDEC910;
 constexpr ::std::ptrdiff_t model_ambient_min = 0x5A118C;
 constexpr ::std::ptrdiff_t set_abs_angles = 0x1E5C50;
 constexpr ::std::ptrdiff_t set_abs_origin = 0x1E5A90;
 } // namespace signatures
-} // namespace hazedumper
+} // namespace hazdumper
```
Comment (JuanUWU69) [inline csgo.hpp:164]: same typo here 😄

File: csgo.hpp (hunk 1 above)
Comment (pachara1337) [reply csgo.hpp:6]: sorry imma fix

File: csgo.hpp (hunk 2 above)
Comment (pachara1337) [reply csgo.hpp:164]: sorry imma fix

**Pull request closed without merging**